### ISSUE TITLE
[menu] Small right-aligned menu items

### ISSUE THREAD

**Issue by ursoforte** [fomantic/Fomantic-UI#2653]
# Bug Report
When I create a "ui.menu" with right aligned, the right-aligned items appear too small. I'm migrating from Semantic-UI 2.4 to Fomantic-UI 2.9, and I noticed this inconsistency.

HTML code example:
```
<div class="ui icon menu small labeled secondary stackable">
            <div class="right menu">
                 <a class="item link active" href="">
                    <i class="paste icon"></i> Action
                </a>
                  .....
            </div>
</div>
```

## Expected result
The normal result is items similar to the main menu. Unless the Fomantic-UI team changed it to default, in which case, it's not a problem.

## Actual result
Small right-aligned menu items and icons.

## Testcase
https://jsfiddle.net/c3Lkrv5z/

## Screenshot (if possible)
![Opera Snapshot_2023-01-13_163941_localhost](https://user-images.githubusercontent.com/578063/212405297-3cb46279-efef-460b-81f9-72628bd0173e.png)

## Version
2.9

**Comment by ursoforte** [comment]
Plus: It can be fixed if you add extra CSS classes in the "right menu", adding:
```
<div class="ui icon menu small labeled secondary stackable">
            <div class="right menu small ui icon labeled secondary">
              ....
             </div>
</div>
```

**Comment by lubber-de** [comment]
Fixed by #2654
